docs/index.adoc

```diff
@@ -30,45 +30,29 @@ anyone.
 TBTC v2 allows anyone to use Bitcoin in the expanding DeFi and Web3 universe
 without a third-party intermediary. 
 
-== System Architecture
+== Overview
 
-=== Wallet States
+=== Deposit
 
-There are six states for a wallet:
+Stakers periodically create 51-of-100 threshold-ECDSA-backed wallets to hold
+frozen Bitcoin assets to maintain account balances. Depositors send Bitcoin to
+the most-recently-created wallet by using pay-to-script-hash (P2SH) or
+pay-to-witness-script-hash (P2WSH) which contains hashed information about the
+depositor’s Ethereum address. The depositor <<depositing,reveals>> their P2(W)SH
+deposit to Ethereum chain. The Bridge listens for these sorts of messages and
+when it gets one, it checks the Bitcoin network to make sure the funds line up.
+If everything checks out, wallet <<sweeping,sweeps>> funds and updates
+Ethereum-side account balances.
```

Comment:
"the" x3?
```suggestion
If everything checks out, the wallet <<sweeping,sweeps>> the funds and updates
the Ethereum-side account balances.
```

Reply:
I am not sure about the second *the* (@beaushinkle ?) but added the other two: 9f853d8204fbcc817b8b316446f78b1a41806977.

Reply:
"the wallet sweeps the funds and updates the balances" is better than "the wallet sweeps funds and updates the balances"

Reply:
I'm not going to block on a debatable "the" though. In we go!